src/main/kotlin/com/pubnub/api/crypto/CryptoModule.kt

```diff
@@ -0,0 +1,110 @@
+package com.pubnub.api.crypto
+
+import com.pubnub.api.PubNubException
+import com.pubnub.api.crypto.cryptor.AesCbcCryptor
+import com.pubnub.api.crypto.cryptor.Cryptor
+import com.pubnub.api.crypto.cryptor.HeaderParser
+import com.pubnub.api.crypto.cryptor.LegacyCryptor
+import com.pubnub.api.crypto.cryptor.ParseResult
+import com.pubnub.api.crypto.data.EncryptedData
+
+class CryptoModule internal constructor(
+    internal val primaryCryptor: Cryptor,
+    internal val cryptorsForEncryptionOnly: MutableList<Cryptor> = mutableListOf(),
```

Comment:
Name is still cryptorsForEncryptionOnly although they're for decryption

Reply:
Good catch. I will fix that.